config/examples/features.yml

```diff
@@ -20,6 +20,8 @@ features: &default
     enabled: true
   email_configuration:
     enabled: false
+  zync:
+    sync_non_oidc_apps: true
```

Comment:
Shouldn't we have `enabled` flag as in all other features? I actually imagined that we would have a feature `zync_sync_all_apps` or `zync_only_oidc_apps` or something like that. My understanding was that we have a single feature and any additional config is only related to that particular feature. Not just adding configuration options for a component...

Reply:
Yeah, I thought about having `enabled`, but I thought it would be confusing - actually related to your other comment.
So, if we keep the feature called `zync`, I would interpret having the `enabled` field as a means of switching whether the whole feature is enabled or disabled, but it's just not true.

I am not sure if `enabled` is required for all, for example `logging` also doesn't have `enabled` field in the config.

> I actually imagined that we would have a feature zync_sync_all_apps or zync_only_oidc_apps or something like that. 

Yeah, I considered that initially too, but then I thought - it's kind of specific, and I think this thing is too small to be considered a Feature. For example, `account_deletion` and `segment_deletion` features have multiple configuration fields.
Imagine, at some point we want to add some other modification to Zync flow. For example, in order to control whether the domain change related events should be pushed to Zync, we may decide to rely on the feature configuration, rather than on whether we're running in SaaS or on-premises. I refer to https://github.com/3scale/porta/pull/3579/files#diff-637d0fc2bb61645722f12335cb20eecbfccfbee12f5e64c2fedaa5b7ad5262beR17-R20 - we could just add a new config for Zync feature, for example, set `do_not_sync_domains: true` for SaaS.

Reply:
But if you're not convinced, I'm OK to change it.